devito/function.py

```diff
@@ -326,9 +327,11 @@ class TimeFunction(Function):
     :param staggered: (Optional) tuple containing staggering offsets.
     :param dtype: (Optional) data type of the buffered data
     :param save: Save the intermediate results to the data buffer. Defaults
-                 to `False`, indicating the use of alternating buffers.
-    :param time_dim: Size of the time dimension that dictates the leading
-                     dimension of the data buffer if :param save: is True.
+                 to `None`, indicating the use of alternating buffers. If
```

Comment:
Nitpicking: Should use double ` around None to get teletype typesetting in sphinx (assuming that's the goal).

Reply:
Fixed